PopPUNK/__main__.py

```diff
@@ -166,6 +176,15 @@ def get_options():
                                               '[default = False]', default=False, action='store_true')
     queryingGroup.add_argument('--accessory-only', help='Use an accessory-distance only model for assigning queries '
                                               '[default = False]', default=False, action='store_true')
+    queryingGroup.add_argument('--assign-lineage', help='Assign to lineages rather than strains - can specify '
+                                              'lineages other than those in the reference database using "--existing-scheme"'
+                                              '[default = False]', default=False, action='store_true')
+
+    # lineage clustering within strains
+    lineagesGroup = parser.add_argument_group('Lineage analysis options')
+    lineagesGroup.add_argument('--R',help='Comma separated list of ranks used in lineage clustering [default = 1]', type = str, default = "1")
+    lineagesGroup.add_argument('--use-accessory',help='Use accessory distances for lineage definitions [default = use core distances]', action = 'store_true', default = False)
+    lineagesGroup.add_argument('--existing-scheme',help='Name of pickle file storing existing lineage definitions', type = str, default = None)
```

Comment:
Don't understand how this would be used, and if it is necessary with some combinations of options

Reply:
Specified 'existing scheme' is to be used with --assign-lineages in effb8909e329c8ac57e1bc9aea09be1088f7021d.